### ISSUE TITLE
Extension won't disable on reload, won't ask me if I want to see comments again

### ISSUE THREAD

**Issue by molly-practice** [estruyf/vscode-hide-comments#1]
Can you help me out with how to get this gosh darned extension to go away once and for all (lol)
I was working through a course and taking lots of notes (in comment form) that was making my files look messy,
so I was looking for a way to hide or collapse them. I found this extension and installed/enabled it. 
The popup came up asking if I want to hide my comments, and I clicked yes

Decided I wanted to see my comments again, so I disabled the extension and reloaded vscode. Comments still transparent.
Uninstalled the extension and rebooted vscode. Still transparent.
When I exit vscode and start it back up, I don't get a popup asking if I want to see my comments again.
I've tried uninstalling and reinstalling this thing 4 times, with various attempts and disabling and reenabling it, and attempts to restart vscode to get the elusive popup, but with no luck

I'm on a windows machine.. is there any way you can just let me know what setting it was that the extension alters so that I can keep getting work done in the meantime? It's really distracting and I'd just like to undo it

Thanks very much

**Comment by estruyf** [comment]
Hi @molly-practice these settings are stored in the settings file of your VSCode project. This you can find in the `.vscode/settigns.json` file in your project. 

You could have also unset it with the `Hide Comments: Show all comments` command in VSCode. Which reverts these settings for you.

**Comment by estruyf** [comment]
Will add it as an enhancement to start a cleanup job.

**Comment by cfoxy17** [comment]
These settings are not in my settings.json file. I have looked in %APPDATA%\Code\User\settings.json and nothing about showing / hiding comments is in this file. How do we fix our vscode installations without reinstalling?

**Comment by estruyf** [comment]
Settings are stored on project level, not on root level. Check the `.vscode/settigns.json` file in your current project.

**Comment by cfoxy17** [comment]
I have done a search of my entire repo and could only find this file in these 2 places:

<project>\node_modules\is-what\.vscode\settings.json
<project>\node_modules\webpack\node_modules\serialize-javascript\.vscode\settings.json

and even if I could find the right file, I'm not sure what value I'm looking for in settings.json

**Comment by estruyf** [comment]
True, it is only the disable that is on the workspace level. In that case, open VSCode, open the command pallet: `ctrl+shift+p` or `cmd+shift+p`, type `Preferences: Open Settings (son)`, and remove the following lines:

```
"editor.tokenColorCustomizations": {
      "comments": "#00000000",
      "textMateRules": [
        {
          "scope": [
            "comment",
            "comment.block",
            "comment.line",
            "comment.line.double-slash"
          ],
          "settings": {
            "foreground": "#00000000"
          }
        }
      ]
    }
```

**Comment by cfoxy17** [comment]
It's fixed now - thanks. It was in "Preferences: Open **Workspace** Settings (JSON)"

**Comment by estruyf** [comment]
There is no trigger to remove the config when the extension gets uninstalled. Closing the issue, but will keep looking for a solution. For now, I have added the steps to the README file.
